### PR TITLE
[v6r14] XROOTSTorage: Fix KeyError exception when checking file existance doe…

### PR DESCRIPTION
**Pull request by andresailer**
…s not work

Add test to check that logic path in putFile as well.

### CODE REVIEW

**Comment by fstagni** [comment]
Review OK. 

Just, completeness:
```
 self.assert_( res['OK'] )
```
is equivalent to
```
 self.assertEqual( True, res['OK'] )
```

**Comment by andresailer** [comment]
Huh? Can you put some context to your "completeness" statement?
I don't see any assert_ in the whole test file, also assert_ is deprecated
https://docs.python.org/2/library/unittest.html#deprecated-aliases
So it would be shorter to use assertTrue, but I wasn't sure if this new unittest version is used everywhere already, so I kept it consistent to what is there.

File: Resources/Storage/XROOTStorage.py [hunk 1]
```diff
@@ -405,7 +405,7 @@ def __putSingleFile( self, src_file, dest_url, sourceSize = 0 ):
       gLogger.debug( errStr, res['Message'] )
 
     # This is true only if the file exists. Then we remove it
-    if res['Value']:
+    if 'Value' in res and res['Value']:
```
Comment (chaen) [inline Resources/Storage/XROOTStorage.py]: if res.get('Value') ?

Comment (andresailer) [inline Resources/Storage/XROOTStorage.py]: Yes, that's better. Thanks.

**Comment by fstagni** [comment]
ah good to know. So, forget my comment.

**Comment by chaen** [comment]
Review OK

**Pull request merged**